client/my-sites/plans-features-main/index.jsx

```diff
@@ -109,6 +114,14 @@ export class PlansFeaturesMain extends Component {
 		return true;
 	}
 
+	isDisplayingPlansNeededForFeature() {
+		const { selectedFeature, selectedPlan, previousRoute } = this.props;
+
+		if ( selectedFeature && selectedPlan && ! previousRoute.startsWith( '/plans/' ) ) {
```

Comment:
Did you add the route check just in case or did you find any case where the user can land in this page coming from other `/plans` router?

Reply:
It's necessary to prevent this issue.

![ezgif-6-6fe344aabfa3](https://user-images.githubusercontent.com/43215253/79474277-1b05b100-7ffe-11ea-8ef3-f5171ae5912f.gif)